### PR TITLE
On- SearchInput, the user sees the map with the zoomed location 

### LINKED ISSUE #27: Add search functionality in the home page search form
As a user, I would like to choose a city close to me and search for available property in that area.

Acceptance criteria

- [ ] Drop down list with major cities and towns
- [ ] ready data with coordinates 
- [ ] functionality to search for the coordinate of the town selected
- [ ] zoomed in map showing the searched town/city

### PR DESCRIPTION
**Pull request by hoslack**
Closes #27

### CODE REVIEW

File: client/src/components/App.js [hunk 1]
```diff
@@ -7,14 +7,45 @@ import Header from './Header';
...
+      selectedOption: '',
+      defaultCenter: { lat: 31.771959, lng: 35.217018 },
+      defaultZoom: 8,
+    };
+    this.handleSearch = this.handleSearch.bind(this);
+  }
+
+  handleSearch(coord, callback) {
+    this.setState({ defaultCenter: coord, defaultZoom: 12 }, () => {
+      callback();
```
Comment (Karyum) [inline client/src/components/App.js:23]: i dont understand this callback do you need it here, can you try this code without having the callback?

File: client/src/components/SearchForm.js [hunk 2]
```diff
@@ -1,11 +1,44 @@
 import React, { Component } from 'react';
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
+const data = require('../citydata');
```
Comment (Karyum) [inline client/src/components/SearchForm.js]: change this from a `require` to `import`

File: client/src/components/App.js (hunk 1 above)
Comment (Karyum) [reply client/src/components/App.js:23]: correct me if im wrong but after the state changes , you want to change to a different page ?

File: client/src/components/SearchForm.js (hunk 2 above)
Comment (Karyum) [reply client/src/components/SearchForm.js]: also use destruction-ring to `{ data  }` so you won't have to do `data.data` [here](https://github.com/FACN3/Israel_Housing/pull/28/files#diff-75814a47c242c9dc4c66e9f653233c55R36)

**Review by Karyum — CHANGES_REQUESTED** [review]

File: client/src/components/App.js (hunk 1 above)
Comment (hoslack) [reply client/src/components/App.js:23]: The callback allows me to redirect the page only after the state has changed. I think it's better that way.

File: client/src/components/SearchForm.js (hunk 2 above)
Comment (hoslack) [reply client/src/components/SearchForm.js]: Yeah. That was a good opportunity to use es6.

Comment (hoslack) [reply client/src/components/SearchForm.js]: Done.

Comment (Karyum) [reply client/src/components/SearchForm.js]: ok ? where is it ?

**Review by Karyum — APPROVED** [review]

**Pull request merged**